app/models/Preset.java

```diff
@@ -37,17 +37,19 @@
     @ManyToOne(cascade = CascadeType.REFRESH)
     public Camera camera;
 
-    @Constraints.Required
-    public int pan;
-
-    @Constraints.Required
-    public int tilt;
+    @JsonIgnore
+    @ManyToOne(cascade = CascadeType.REFRESH)
+    public Script script;
 
-    @Constraints.Required
-    public int zoom;
+    /**
+     * Used to link to a real world preset.
+     */
+    public int realPresetId;
```

Comment:
Real world preset? Let's not use the 'dummy'/'real' terminology. Better is something like 'linkedCameraPreset'.